ads/google/a4a/test/test-utils.js

```diff
@@ -294,10 +294,29 @@ describe('Google A4A utils', () => {
         const impl = new MockA4AImpl(elem);
         noopMethods(impl, doc, sandbox);
         impl.win.AMP_CONFIG = impl.win.AMP_CONFIG || {};
-        impl.win.AMP_CONFIG.canary = false;
+        impl.win.AMP_CONFIG.type = 'control';
         return fixture.addElement(elem).then(() => {
           return googleAdUrl(impl, '', 0, [], []).then(url1 => {
-            expect(url1).to.not.match(/art=2/);
+            expect(url1).to.match(/art=1/);
+          });
+        });
+      });
+    });
+    it('should not have `art` parameter', () => {
+      return createIframePromise().then(fixture => {
+        setupForAdTesting(fixture);
+        const doc = fixture.doc;
+        doc.win = window;
+        const elem = createElementWithAttributes(doc, 'amp-a4a', {
+          'type': 'adsense',
+          'width': '320',
+          'height': '50',
+        });
+        const impl = new MockA4AImpl(elem);
+        noopMethods(impl, doc, sandbox);
+        return fixture.addElement(elem).then(() => {
+          return googleAdUrl(impl, '', 0, [], []).then(url1 => {
+            expect(url1).to.not.match(/art/);
```

Comment:
Better to match /art=/ ? Really though, it would be better to use a narrower regex for all url param matches, e.g., something like /[&?]xxx=/.

Reply:
Done.